pipelines/build/openjdk_build_pipeline.groovy

```diff
@@ -144,11 +145,56 @@ def sign(config) {
     }
 }
 
+def listArchives() {
+    return sh(
+            script: """find workspace/target/ | egrep '.tar.gz|.zip'""",
+            returnStdout: true,
+            returnStatus: false
+    )
+            .trim()
+            .split('\n')
+}
+
+def writeMetadata(config, filesCreated) {
+    def buildMetadata = [
+            os     : config.os,
+            arc    : config.arch,
+            variant: config.variant,
+            version: config.javaVersion,
+            tag    : config.parameters.TAG
+    ]
+
+    /*
+    example data:
+    {
```

Comment:
I wonder if this needs to be in a top-level block.  What if for example, we added a `build_configure` section with all of the `bash configure` output or the versions of all of the O/S native libs we build with etc....